app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SessionsController < ApplicationController
+  def create
+    @user = User.find_or_create_from_auth_hash(request.env["omniauth.auth"])
```

Comment:
The word 'request' was needed and would not hang on to the ouath hash unless it was used